### DISCUSSION TITLE
essay/2024/11/04/ownership

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#242] [Comments]
# essay/2024/11/04/ownership

직원이 회사에서 주인 의식을 가질 수 있을까? 주인이 아닌데 주인처럼 행동한다? 바보 같고 우스워 보이는 일입니다. 특히나 주식도 없고 스톡옵션도 없다면요.

https://jeho.page/essay/2024/11/04/ownership.html

**Comment by yoonjong-park** [comment]
아마도 대표가 비젼을 보여주었고, 그에 설득당할 수 있었기 때문일 겁니다.

애플이나 구글같은 기업, 혹은 굉장히 좋은 서비스나 솔루션을 
가졌거나 가지려고하는 회사에 속한 직원만이 그럴 수 있는 것 같습니다.

**Reply by BenjaminKim** [reply]
네, 특별한 회사만이 그럴 수 있는 것 같습니다. 그런 회사에 다녀야 인생이 행복하다고 생각하고요. 😁
